app/models/incomeSourceDetails/PropertyDetailsModel.scala

```diff
@@ -22,7 +22,9 @@ import play.api.libs.json.{Json, OFormat}
 
 case class PropertyDetailsModel(incomeSourceId: Option[String],
                                 accountingPeriod: Option[AccountingPeriodModel],
-                                firstAccountingPeriodEndDate: Option[LocalDate])
+                                firstAccountingPeriodEndDate: Option[LocalDate],
+                                incomeSourceType: Option[String],
```

Comment:
* please add enumeration representing set of incomeSourceTypes and use instead of Option[String];
* tradingStartDate - must be Option[LocalDate] assuming on the naming convention used for the field;